### ISSUE TITLE
Upgrading to webpack 4 breaks the plugin

### ISSUE THREAD

**Issue by Havunen** [webpack-contrib/eslint-loader#217]
Webpack 4 has removed `this.options` See blog post here:
https://medium.com/webpack/webpack-4-migration-guide-for-plugins-loaders-20a79b927202

This line breaks:
https://github.com/webpack-contrib/eslint-loader/blob/master/index.js#L148

It probably should use `query` property for webpack 4 but I'm not sure.

**Comment by renarsvilnis** [comment]
Can confirm the issue.

```
ERROR in ./src/index.js
Module build failed: TypeError: Cannot read property 'eslint' of undefined
    at Object.module.exports (/path/to/my/project/node_modules/eslint-loader/index.js:148:18)
 @ multi babel-polyfill react-hot-loader/patch ./
```

**Comment by MoOx** [comment]
Dup of #204

**Comment by MoOx** [comment]
Fixed in 2.0.0
